doc/releases/changelog-dev.md

```diff
@@ -55,6 +55,9 @@
 
 <h3>Deprecations 👋</h3>
 
+* `qml.grad` and `qml.jacobian` now explicitly raise errors if trainable parameters are integers.
+  [(#4836)](https://github.com/PennyLaneAI/pennylane/pull/4836)
+
```

Comment:
Would this be more of an improvement or bugfix than a deprecation?

Reply:
Eh... oops?

Reply:
Fixed :+1: